pyhtools/attackers/web/email_scraper.py

```diff
@@ -0,0 +1,74 @@
+from bs4 import BeautifulSoup
+import requests, re, lxml
+from requests import exceptions
+from urllib import parse
+from collections import deque
+from pyhtools.UI.colors import BRIGHT_RED, BRIGHT_WHITE, BRIGHT_YELLOW
+
+##----------------Author: astralm0nke on GitHub-----------------##
+class EmailScraper:
+    USER_URL = str(input('[+] Specify Target URL to Scan: '))
+    USER_COUNT = int(input('[+] Specify Count Limit: '))
+    COUNT = 0
+    URLS = deque([USER_URL])
+    SCRAPPED_URLS = set()
+    EMAILS = set()
+    
+# Main function to scrape emails
+    def scrape(self):
```

Comment:
Let's remove comment and add docstring in this [format](https://github.com/dmdhrumilmistry/pyhtools/blob/deadf6d0aa04d0e3666cc41e2aaf5527b126f184/pyhtools/attackers/web/utils.py#L39) so others can know how to use this function.